content/patterns/button/button-pattern.html

```diff
@@ -14,92 +14,93 @@
-            For example, a button labelled mute in an audio player could indicate that sound is muted by setting the pressed state true.
-            <strong>Important:</strong> it is critical the label on a toggle does not change when its state changes.
-            In this example, when the pressed state is true, the label remains &quot;Mute&quot; so a screen reader would say something like &quot;Mute toggle button pressed&quot;.
-            Alternatively, if the design were to call for the button label to change from &quot;Mute&quot; to &quot;Unmute,&quot; the aria-pressed attribute would not be needed.
+            トグルボタン：オフ（押されていない状態）とオン（押された状態）の2つの状態を持つボタン。
+            ボタンがトグルボタンであることを支援技術に伝えるには、<a href="#aria-pressed" class="state-reference">aria-pressed</a> 属性に値を指定します。
+            例えば、オーディオプレーヤーのミュートというラベルのボタンは、押された状態を true に設定することで、音がミュートされていることを示すことができます。
+            <strong>重要：</strong>トグルボタンのラベルは、その状態が変化しても変化しないことが重要です。
+            この例では、押された状態が true のとき、ラベルは &quot;ミュート&quot; のままなので、スクリーンリーダーは "ミュート トグルボタン 押されています" のように言うでしょう。
+            そうではなく、ボタンのラベルが "ミュート" から "ミュート解除" に変わるようなデザインであれば、aria-pressed 属性は必要ありません。
           </li>
-          <li>Menu button: as described in the <a href="../menu-button/menu-button-pattern.html">menu button pattern</a>, a button is revealed to assistive technologies as a menu button if it has the property <a href="#aria-haspopup" class="property-reference">aria-haspopup</a> set to either <code>menu</code> or <code>true</code>.</li>
+          <li>メニューボタン： <a href="../menu-button/menu-button-pattern.html">メニューボタン パターン</a> で説明されているように、ボタンが支援技術にメニューボタンとして認識されるためには、<a href="#aria-haspopup" class="property-reference">aria-haspopup</a> プロパティが <code>menu</code> 又は <code>true</code> に設定されている必要があります。</li>
         </ul>
         <div class="note">
-          <h3>Note</h3>
+          <h3>ノート</h3>
           <p>
-            The types of actions performed by buttons are distinctly different from the function of a link (see <a href="../link/link-pattern.html">link pattern</a>).
-            It is important that both the appearance and role of a widget match the function it provides.
-            Nevertheless, elements occasionally have the visual style of a link but perform the action of a button.
-            In such cases, giving the element role <code>button</code> helps assistive technology users understand the function of the element.
-            However, a better solution is to adjust the visual design so it matches the function and ARIA role.
+            ボタンによって実行されるアクションの種類は、リンクの機能（ <a href="../link/link-pattern.html">リンクパターン</a> を参照）とは明確に異なります。
+            ウィジェットの外観及び役割は、それが提供する機能と一致することが重要です。
+            それにもかかわらず、要素がリンクの視覚的なスタイルを持ちながら、ボタンのアクションを実行することがあります。
+            このような場合、要素に <code>button</code> の役割を与えることは、支援技術利用者が要素の機能を理解するのに役立ちます。
+            しかし、より良い解決策は、機能と ARIA の役割が一致するように視覚デザインを調整することです。
```

Comment:
ARIAの文脈で「role」を使う時は「役割」ではなく「ロール」としておきたいです。
```suggestion
            ウィジェットの外観及びロールは、それが提供する機能と一致することが重要です。
            それにもかかわらず、要素がリンクの視覚的なスタイルを持ちながら、ボタンのアクションを実行することがあります。
            このような場合、要素に <code>button</code> ロールを与えることは、支援技術利用者が要素の機能を理解するのに役立ちます。
            しかし、より良い解決策は、機能と ARIA ロールが一致するように視覚デザインを調整することです。
```